code/game/objects/effects/decals/Cleanable/tar.dm

```diff
@@ -0,0 +1,42 @@
+/obj/effect/decal/cleanable/tar
+	gender = PLURAL
+	name = "tar"
+	desc = "A sticky substance."
+	icon_state = "tar2"
+	density = FALSE
+	opacity = FALSE
+	var/turf/simulated/target
```

Comment:
I doubt there is a need for this variable, you should be able to just use the `get_turf(src)` to get the location of the tar. I don't believe the tar ever moves, but perhaps you should add a safety for that, like

```
tar_typepath/Moved(atom/OldLoc, Dir, Forced)
	. = ..()
	oldLoc.slowdown -= 10
    loc.slowdown += 10
```

Reply:
Using get_turf(scr) in place of target throws errors - target should be fine?

Reply:
And trying to deal with the tar possibly ending up on space turfs and such seems like a lot of hassle for something that should be impossible.

Reply:
Could you show the errors it throws?

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/18598676/3a476caf-7447-4f60-8751-f5eea5e462a0)
Seems like I'd have to define a variable for them, which I'm doing already, so...

Reply:
just use a local variable
var/turf/T = get_turf(src)
T.slowdown += or -= 10

Reply:
Switching over to that meant that I couldn't check if(target), which introduced a bug. Target was also really useful for implementing Moved()